### DISCUSSION TITLE
Empty values when binding to Eloquent model

### DISCUSSION THREAD

**Discussion by emildayan** [livewire/livewire#3548] [7. Bugs (archived)]
Cannot update company.email to an empty value. As soon as it becomes empty, it resets to default value (filled).

`use AuthorizesRequests;

    public Company $company;
    protected      $rules = [
        'company.name'  => ['required', 'string'],
        'company.email' => ['nullable', 'sometimes', 'email'],
    ];

    public function mount(Company $company)
    {
        $this->authorize('view', $company);
        $this->company = $company;
    }

    public function save()
    {
        $this->validate();
        $this->company->save();
        $this->company->wasUpdated();
    }

    public function render()
    {
        return view('livewire.company.details');
    }

    ------

    <form wire:submit.prevent="save" class="form">
        <x-input.text-simple wire:model="company.name" name="company.name" required/>
        <x-input.text-simple wire:model="company.email" name="company.email"/>
        
        <button wire:loading.class="loading" wire:target="save" type="submit">Save</button>
    </form>`

**Comment by joshhanley** [comment]
@emildayan I've just done a test (using a user model) with the rules you have here, and I can set the users email to empty and hit save, and it validates and saves without any issues.

What version of Livewire are you using?

Is there any steps in particular you are following that is causing it to happen?
